### PR TITLE
Black-Box attack tutorial

### PR DESCRIPTION
**Pull request by npapernot**
This adds a tutorial for this paper: https://arxiv.org/abs/1602.02697

No changes made to the cleverhans module itself

### CODE REVIEW

File: tutorials/mnist_blackbox.py [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+              Dense(nb_classes),
+              Activation('softmax')]
+
+    for layer in layers:
+        model.add(layer)
+
+    return model
+
+
+def jacobian_augmentation(sess, x, X_sub_prev, Y_sub, grads):
```
Comment (goodfeli) [inline tutorials/mnist_blackbox.py]: This might be worth moving into the main cleverhans module sometime, but no need to hold up this PR

**Comment by goodfeli** [comment]
Looks good to merge once the pep8 test runs

File: tutorials/mnist_blackbox.py (hunk 1 above)
Comment (npapernot) [inline tutorials/mnist_blackbox.py]: good point I opened an issue to remember this: #78

**Comment by npapernot** [comment]
I added the missing init files by committing directly to the master branch so we don't mix things up with this PR in the future (b1bac37 )

**Pull request merged**